### PR TITLE
[Feature] Add scripts and documents for helm chart development and testing

### PR DESCRIPTION
**Pull request by AaronLinOops**
### What is the purpose of the change
Add scripts and documents for helm chart development and testing

### Brief change log
- Add a document
- Add test utils scripts to support create KinD cluster and test charts

### Verifying this change
(Please pick either of the following options)  
tools with no test case

### Does this pull request potentially affect one of the following parts:
- Dependencies (does it add or upgrade a dependency): (no)
- Anything that affects deployment: (no)
- The MGS(Microservice Governance Services), i.e., Spring Cloud Gateway, OpenFeign, Eureka.: (no)

### Documentation
- Does this pull request introduce a new feature? (yes)
- If yes, how is the feature documented? (docs)

### CODE REVIEW

**Review by jacktao007 — APPROVED** [review]
LGTM

File: linkis-dist/helm/scripts/resources/kind-cluster.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+kind: Cluster
```
Comment (legendtkl) [inline linkis-dist/helm/scripts/resources/kind-cluster.yaml:1]: ASF header is needed

File: linkis-dist/helm/scripts/resources/mysql.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (legendtkl) [inline linkis-dist/helm/scripts/resources/mysql.yaml:1]: ASF header is needed.

**Review by legendtkl — CHANGES_REQUESTED** [review]
LGTM with some comments.

File: linkis-dist/helm/README.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
+Helm charts for Linkis
```
[the remark this answers is not in the document]
Comment (AaronLinOops) [reply linkis-dist/helm/README.md:1]: add [License](https://www.apache.org/licenses/LICENSE-2.0.html) as this is a markdown doc

File: linkis-dist/helm/README.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+## Destroy the local cluster
+```shell
+# Option 1: delete the helm release only
+$> helm delete --namespace linkis linkis-demo 
+
+# Option 2: destroy the KinD cluster, no need to delete
+# the helm release first
+$> kind delete cluster --name test-helm
+```
```
[the remark this answers is not in the document]
Comment (AaronLinOops) [reply linkis-dist/helm/README.md]: good catch

File: linkis-dist/helm/README.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
+Helm charts for Linkis 
+==========
+
+# Pre-requisites
+* [Kubernetes](https://kubernetes.io/docs/setup/), minimum version v2.21.0+
```
[the remark this answers is not in the document]
Comment (AaronLinOops) [reply linkis-dist/helm/README.md]: good catch

**Pull request merged**